index.html

```diff
@@ -143,6 +143,44 @@ <h2 class="section-title" id="add-task-header">Add New Task</h2>
         </section>
         <section class="task-list">
           <h2 class="section-title">My Task List</h2>
+          <div class="sort-section">
+            <label for="sort-tasks">Sort tasks by :&nbsp; </label>
+            <select name="sort-tasks" id="sort-tasks">
+              <option value="not-sorted">not sorted</option>
```

Comment:
```suggestion
              <option value="">not sorted</option>
```

Reply:
Follow same namings for values. 
Either:
`Not sorted` and `All tasks`
or
`not sorted` and `all tasks`

Reply:
yes ,I got it